electron/src/js/lib/googleAuth.ts

```diff
@@ -66,31 +68,36 @@ const getAccessToken = (scopes, clientId, clientSecret) => {
         redirect_uri: 'urn:ietf:wg:oauth:2.0:oob',
       });
 
-      request.post(
-        'https://accounts.google.com/o/oauth2/token',
-        {
-          body: data,
-          headers: {
-            Accept: 'application/json',
-            'Content-Type': 'application/x-www-form-urlencoded',
-          },
+      const requestConfig = {
+        body: data,
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/x-www-form-urlencoded',
         },
-        (error, response, body) => (error ? reject(error) : resolve(JSON.parse(body)))
-      );
+      };
+
+      const requestUrl = 'https://accounts.google.com/o/oauth2/token';
+
+      request.post(requestUrl, requestConfig, (error, response, body) => {
+        if (error) {
+          return reject(error);
+        }
+
+        const result = JSON.parse(body) as GoogleAccessTokenResult;
+        return resolve(result);
+      });
```

Comment:
Can easily be turned into a ternary

```suggestion
        return error ? reject(error) : resolve(JSON.parse(body) as GoogleAccessTokenResult);
      });
```

Reply:
Yes but that makes it really unreadable.

Reply:
As discussed f2f, will leave it like it is.